api/kroxylicious-filter-api/pom.xml

```diff
@@ -34,16 +34,19 @@
         <dependency>
             <groupId>io.kroxylicious</groupId>
             <artifactId>kroxylicious-api</artifactId>
+            <scope>provided</scope>
         </dependency>
 
         <!-- third party dependencies - runtime and compile -->
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-annotations</artifactId>
+            <scope>provided</scope>
```

Comment:
Maybe this dependency should move to kroxylicious-api as it's more for the BaseConfig

Reply:
Fixed in 8a69e1b6773e57416437670ec4a5080b16bb6e4d